src/ukify/ukify.py

```diff
@@ -449,7 +449,26 @@ class PeError(Exception):
 
 def pe_add_sections(uki: UKI, output: str):
     pe = pefile.PE(uki.executable, fast_load=True)
-    assert pe.FILE_HEADER.PointerToSymbolTable != 0 or len(pe.__data__) % pe.OPTIONAL_HEADER.FileAlignment == 0
+
+    # Old stubs might have been stripped, leading to unaligned raw data values, so let's fix them up here.
+    for i in range(len(pe.sections)):
+        oldp = pe.sections[i].PointerToRawData
+        oldsz = pe.sections[i].SizeOfRawData
+        pe.sections[i].PointerToRawData = round_up(pe.sections[i].PointerToRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        pe.sections[i].SizeOfRawData = round_up(pe.sections[i].SizeOfRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        padp = pe.sections[i].PointerToRawData - oldp
+        padsz = pe.sections[i].SizeOfRawData - oldsz
+
+        for j in range(i + 1, len(pe.sections)):
+            pe.sections[j].PointerToRawData += padp + padsz
+
+        pe.__data__ = pe.__data__[:oldp] + padp * b'\0' + pe.__data__[oldp:oldp+oldsz] + padsz * b'\0' + pe.__data__[oldp+oldsz:]
```

Comment:
There is `pe.set_data_bytes()`, we should probably use that.

Also, is `bytearray(padding)` more pythonic?

Reply:
Switched to `bytearray()`, works fine

Let's leave the `set_data_bytes()` thing for a subsequent PR since we need to access it anyway

Reply:
What does the type have to do with how to express creating a padding? You can just as well use `bytes(padding)`. Also, internally, pefile uses `bytearray` once the data is modified.

Reply:
`_set_data_bytes()` overrides data, it doesn't insert new data, so it doesn't work in this case.